README.md

```diff
@@ -1 +1,289 @@
+BLUEFOX_TEMPLATE_ID_ADMIN_INVITATION=<bluefox_template_id> # Bluefox Template ID for admin invitation
+BLUEFOX_TEMPLATE_ID_ACCOUNT_FINALIZE_REGISTRATION=<bluefox_template_id> # Bluefox Template ID for account registration finalization
+BLUEFOX_TEMPLATE_ID_ACCOUNT_FORGOT_PASSWORD=<bluefox_template_id> # Bluefox Template ID for account forgot password
+BLUEFOX_TEMPLATE_ID_ACCOUNT_INVITATION=<bluefox_template_id> # Bluefox Template ID for account invitation
+BLUEFOX_TEMPLATE_ID_ACCOUNT_LOGIN_SELECT=<bluefox_template_id> # Bluefox Template ID for account login selection
+BLUEFOX_TEMPLATE_ID_ACCOUNT_VERIFY_EMAIL=<bluefox_template_id> # Bluefox Template ID for account email verification
+BLUEFOX_TEMPLATE_ID_ACCOUNT_CREATE_PASSWORD=<bluefox_template_id> # Bluefox Template ID for account password creation
+
+TEST_STATIC_SERVER_URL=<test_static_server_url> # URL for testing static server
+CDN_BASE_URL=<cdn_base_url> # Base URL for your CDN
+
+AWS_BUCKET_PATH=<aws.bucket.path> # Path to the AWS bucket
+AWS_BUCKET_NAME=<your_aws_bucket_name> # Name of the AWS S3 bucket
+AWS_FOLDER_NAME=<your_aws_folder_name> # Folder name in the AWS S3 bucket
+AWS_REGION=<your_aws_region> # AWS region for the S3 bucket
+AWS_ACCESS_KEY_ID=<your_aws_access_key_id> # AWS access key ID
+AWS_SECRET_ACCESS_KEY=<your_aws_secret_access_key> # AWS secret access key
+
+ALPHA_MODE=false # Enable alpha mode (true/false)
+MAX_FILE_SIZE=5242880 # Maximum file upload size in bytes (e.g., 5MB)
+
+GOOGLE_CLIENT_ID=<your_google_client_id> # Google OAuth client ID
+GOOGLE_CLIENT_SECRET=<your_google_client_secret> # Google OAuth client secret
+
+MICROSOFT_CLIENT_ID=<your_microsoft_client_id> # Microsoft OAuth client ID
+MICROSOFT_CLIENT_SECRET=<your_microsoft_client_secret> # Microsoft OAuth client secret
+
+GITHUB_CLIENT_ID=<your_github_client_id> # GitHub OAuth client ID
+GITHUB_CLIENT_SECRET=<your_github_client_secret> # GitHub OAuth client secret
+
+```
+
+### Customization
+
+The `mua-backend` allows users to customize behavior after specific actions, such as creating or deleting accounts. Pass a `hooks` object with functions to override default behaviors:
+
+```javascript
+
+MuaBackend({
+  apiServer,
+  AccountModel,
+  UserModel,
+  SystemAdminModel,
+  hooks: {
+    deleteAccount: {
+      post: (params) => {
+        // Custom logic after deleting an account
+      },
+    },
+    createAccount: {
+      post: (params) => {
+        // Custom logic after creating an account
+      },
+    },
+    createNewUser: {
+      post: (params) => {
+        // Custom logic after creating a new user
+      },
+    },
+    updateUserEmail: {
+      post: (params) => {
+        // Custom logic after updating a user's email
+      },
+    },
+  },
+})
+
+```
+
+Each hook accepts an object with a `post` method that executes custom logic after the associated action is completed.
+
+## API Routes
+
+The following table outlines the automatically generated routes and their purposes:
```

Comment:
Here is the time to write down, that devs using mua won't really need to bother with these routes, because they work seamlessly with the mua-frontend.